shared/src/schema/EntityTypeAttribute.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Attributes of a entity type.
+ * Contains descriptive information about an entity type.
+ */
+export interface EntityTypeAttribute {
+  /**
+   * Name of the attribute
+   */
+  name: string;
+  /**
+   * Possible data types of the attribute.
+   */
+  types: ['Long' | 'String' | 'StringArray' | string];
```

Comment:
What other types are possible? Maybe list a (relatively) complete set here.

Reply:
The [DB call](https://neo4j.com/docs/operations-manual/current/reference/procedures/#procedure_db_schema_nodetypeproperties) only returns a list of strings.
These strings don't match the [possible data types](https://neo4j.com/docs/cypher-manual/current/syntax/values/).
These values are enough for not but have to be increased in the future (Issue #85)